packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4562,3 +4563,70 @@ export class AppParamsGet extends Op {
 		}
 	}
 }
+
+export class AcctParamsGet extends Op {
+	readonly interpreter: Interpreter;
+	readonly line: number;
+	readonly field: string;
+	/**
+	 * Asserts 1 arguments are passed.
+	 * @param args Expected arguments: [] // none
+	 * @param line line number in TEAL file
+	 * @param interpreter interpreter object
+	 */
+	constructor(args: string[], line: number, interpreter: Interpreter) {
+		super();
+		this.line = line;
+		this.interpreter = interpreter;
+		assertLen(args.length, 1, line);
+
+		if (
+			!AcctParamQueryFields[args[0]] ||
+			AcctParamQueryFields[args[0]].version > interpreter.tealVersion
+		) {
+			throw new RuntimeError(RUNTIME_ERRORS.TEAL.UNKNOWN_ACCT_FIELD, {
+				field: args[0],
+				line: line,
+				tealV: interpreter.tealVersion,
+			});
+		}
+
+		this.field = args[0];
+	}
+
+	execute(stack: Stack<StackElem>): void {
+		this.assertMinStackLen(stack, 1, this.line);
+
+		const acctAddress = this.assertAlgorandAddress(stack.pop(), this.line);
+
+		// get account from current context
+		const accountInfo = this.interpreter.getAccount(acctAddress, this.line, false);
+
+		let value: StackElem = 0n;
+		switch (this.field) {
+			case "AcctBalance": {
+				value = BigInt(accountInfo.balance());
+				break;
+			}
+			case "AcctMinBalance": {
+				value = BigInt(accountInfo.minBalance);
+				break;
+			}
+			case "AcctAuthAddr": {
+				if (accountInfo.getSpendAddress() === accountInfo.address) {
+					value = ZERO_ADDRESS;
+				} else {
+					value = Buffer.from(decodeAddress(accountInfo.getSpendAddress()).publicKey);
+				}
+				break;
+			}
```

Comment:
add a default case, which will return an error , something like `'unknown account parameter "%s"'`

Reply:
We can't create Opcode class if the parameter invalid(will throw error) so we don't need handle this case.